### DISCUSSION TITLE
yarn build

### DISCUSSION THREAD

**Discussion by aniwn88480857** [umami-software/umami#1289] [General]
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Error: Get config: Unable to establish a connection to query-engine-node-api library

Prisma CLI Version : 3.15.2
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "build-db-client" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "build-db" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

You may need to install some additional packages for your OS. See https://github.com/prisma/prisma/issues/13717.

The same failure, I'm so annoying, it's been two days, I still can't get this thing. Replaced three computers. gradually collapse

**Comment by mikecao** [comment]
You need to provide more information. OS, node version, etc. What did you try, what did you install?

**Comment by aniwn88480857** [comment]
$ prisma generate
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Error: Get Config: Unable to establish a connection to query-engine-node-api library.
Details: Unable to require(`D:\sx\um\umami-master\node_modules\prisma\query_engine-windows.dll.node`)
 The specified module could not be found.
\\?\D:\sx\um\umami-master\node_modules\prisma\query_engine-windows.dll.node

Prisma CLI Version : 4.0.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "build-db-client" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "build-db" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.


windows 10:
node : 16
Prisma CLI Version : 4.0.0
